system_tests/staker_test.go

```diff
@@ -464,3 +466,480 @@ func stakerTestImpl(t *testing.T, faultyStaker bool, honestStakerInactive bool)
 func TestStakersCooperative(t *testing.T) {
 	stakerTestImpl(t, false, false)
 }
+
+func TestFastConfirmation(t *testing.T) {
```

Comment:
For these tests, we should increase the rollup config's confirmation day to something like a day to ensure that fast confirmation is being used.

Reply:
Done.